recipes/pyqt5-sip/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyqt5-sip" %}
+{% set version = "12.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyQt5_sip-{{ version }}.tar.gz
+  sha256: 0a34b6596bdd28d52da3a51fa8d9bb0b287bcb605c2512aa3251b9028cc71f4d
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+#--> This package can not be tested by import this way!
+# TODO: find another way to test it 
+#  imports:
+#    - pyqt5_sip
```

Comment:
Is this missing dependencies?

Reply:
more like **wrong** dependencies ...
<details>

```sh
(spyder) nerohmot@THOR:~$ conda list qt
# packages in environment at /home/nerohmot/conda-envs/spyder:
#
# Name                    Version                   Build  Channel
pyqt                      5.12.3           py37h8685d9f_3    conda-forge
pyqt5-sip                 4.19.18                  pypi_0    pypi
pyqtchart                 5.12                     pypi_0    pypi
pyqtwebengine             5.12.1                   pypi_0    pypi
pytest-qt                 3.3.0                      py_0    conda-forge
qt                        5.12.5               hd8c4c69_1    conda-forge
qtawesome                 0.7.2              pyh9f0ad1d_0    conda-forge
qtconsole                 4.7.5              pyh9f0ad1d_0    conda-forge
qtpy                      1.9.0                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
(spyder) nerohmot@THOR:~$ ipython
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:57:50) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from PyQt5 import sip                                                                                                                         

In [2]: from PyQt5 import QtChart                                                                                                                     

In [3]: from PyQt5 import QtWebEngine                                                                                                                 

In [4]:               
```

</details>

The `pyqt` conda package provides the `PyQt5` python package, but as one can see in [pyqt's feedstock](https://github.com/conda-forge/pyqt-feedstock/blob/21c77063f98956797b912a034b78383183f83f54/recipe/meta.yaml#L18) `PyQt5` already holds the `sip`, `QtChart` and `QtWebEngine` packages (double confirmed above in the details).

There is thus **no need** for `pyqt5-sip` (and the other 2) to get a feedstock, rather, we need to find which package(s?) wrongfully require `pyqt5-sip`, `pyqtchart` and/or `pyqtwebengine` and correct those dependencies there.

So, best to close this PR. :disappointed:

@ccordoba12 @goanpeca